windows/win_regedit.ps1

```diff
@@ -118,8 +118,8 @@ else
 
 }
 
-if($registryDataType -eq "binary" -and $registryData -ne $null) {
-   $registryData = Convert-RegExportHexStringToByteArray($registryData)
+if($registryDataType -eq "binary" -and $registryData -ne $null -and $registryData.GetType().Name -eq 'String') {
```

Comment:
Not going to hold up the merge for this, but any reason you're not using -is [String] here? I see people using this GetType().Name -eq 'Whatever' pattern all over the place and wonder why... :) Someday I'll go fix them all...